src/combine-results.ts

```diff
@@ -59,7 +59,7 @@ async function run(): Promise<void> {
         const output = await getExecOutput("wc", ["-l", csvDest]); // TODO: preferably we would do this during results interpretation
         const results = parseInt(output.stdout.trim()) - 2;
```

Comment:
Also, why not `const results = parseInt(output.stdout.trim()) - `;`

Reply:
This is the correct change! Thanks. It's a translated version of https://github.com/dsp-testing/qc-controller/blob/767b54fb9d8f876cc411e98330392ddbe4ca376c/.github/workflows/codeql-query.yml#L104

So I'm asking how many lines there are in `csvDest` excluding the header. `tail -n +2` starts counting from the second line, which is obviously not the same as subtracting 2 :facepalm: